spec/main-process/appwindow-spec.js

```diff
@@ -0,0 +1,240 @@
+'use strict';
+
+var assert = require('assert');
+var sinon = require('sinon');
+var SandboxedModule = require('sandboxed-module');
+
+describe('AppWindow', function () {
+  var sandbox;
+  var AppWindow;
+  var fakeDialog;
+  var fakeBrowserWindow;
+  var fakeShell;
+  var appWindow;
+  var fakeKillChildProcess;
+
+  beforeEach(function () {
+    sandbox = sinon.sandbox.create();
+
+    fakeBrowserWindow = sandbox.stub().returns({
```

Comment:
You can replace the `fakeBrowserWindow` with this, soon as #66 get merged.
```js
dialog: remote.require('dialog')
```

Note, you have to require `remote` once per file. For detail, check out the electron documentation https://github.com/atom/electron/blob/master/docs/api/remote.md

```js
var remote = require('remote');
```